pykafka/handlers.py

```diff
@@ -86,6 +94,22 @@ def spawn(self, target, *args, **kwargs):
         return t
 
 
+class GEventHandler(Handler):
+    """A handler that uses a greenlet to perform its work"""
+    Queue = JoinableQueue
+    Event = GEEvent
+    Lock = GERLock  # fixme
```

Comment:
This is an `RLock` and it should be a regular `Lock`, but gevent doesn't seem to have one?